pkg/kubelet/cm/topologymanager/fake_topology_manager_test.go

```diff
@@ -5,8 +5,8 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
-
+	http://www.apache.org/licenses/LICENSE-2.0
+	
```

Comment:
There seems to be alot of these white-space changes.
It would be good to have these removed from each commit (and isolated to their own commit) so as not to distract from the purpose of the commit.

Reply:
It was unnecessarily introduced in refactoring. I squashed white-space changes to original commits.